src/vl-rich-data-table.js

```diff
@@ -54,31 +55,42 @@ export class VlRichDataTable extends VlElement(HTMLElement) {
                 @import "/node_modules/vl-ui-icon/dist/style.css";
                 @import "/node_modules/vl-ui-button/dist/style.css";
                 @import "/node_modules/vl-ui-data-table/dist/style.css";
+                @import "/node_modules/vl-ui-modal/dist/style.css";
```

Comment:
Het kan geen kwaad, maar een modal is geen native element dus heeft geen CSS import nodig. Best verwijderen.